packages/@aws-cdk/aws-lambda-nodejs/lib/bundling.ts

```diff
@@ -37,7 +42,7 @@ export class Bundling implements cdk.BundlingOptions {
    * esbuild bundled Lambda asset code
    */
   public static bundle(options: BundlingProps): AssetCode {
-    return Code.fromAsset(path.dirname(options.depsLockFilePath), {
+    return Code.fromAsset(options.projectRoot ?? path.dirname(options.depsLockFilePath), {
```

Comment:
```suggestion
    return Code.fromAsset(options.projectRoot, {
```

Reply:
@relm923 can you please apply this suggestion?

Reply:
Done